### PR TITLE
JWT support for user sessions. Basic authentication function wrapper …

### PR DESCRIPTION
**Pull request by Tsintsir**
I haven't actually annotated any functions that could potentially use authentication.

### CODE REVIEW

File: src/db.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from pymongo import MongoClient
 from pbkdf2 import PBKDF2
 import subprocess
+import jwt
```
Comment (shawkinsl) [inline src/db.py]: a HA! I found the culprit for why the tests broke!

When you add a new external library, make sure to put it in requirements.txt. I got this one, but watch out for that one in the future as well!

**Comment by shawkinsl** [comment]
Made a few minor changes, but looks good! +1

**Comment by Tsintsir** [comment]
I don't see `acceptance-tests beta` on Jenkins and the details link returns a 404. Should I merge?

**Comment by shawkinsl** [comment]
acceptance-tests beta actually shouldn't have even run on this branch, you can ignore it. This is fine to merge

**Pull request merged**